src/transformers/convert_slow_tokenizer.py

```diff
@@ -1019,12 +1019,23 @@ def converted(self) -> Tokenizer:
 
         tokenizer.pre_tokenizer = pre_tokenizers.ByteLevel(add_prefix_space=ot.add_prefix_space)
         tokenizer.decoder = decoders.ByteLevel()
+        '''
         tokenizer.post_processor = processors.RobertaProcessing(
             sep=(ot.sep_token, ot.sep_token_id),
             cls=(ot.cls_token, ot.cls_token_id),
             add_prefix_space=ot.add_prefix_space,
             trim_offsets=True,  # True by default on Roberta (historical)
         )
+        '''
+
+        tokenizer.post_processor = processors.TemplateProcessing(
+            single="<s> $A </s>",
+            pair="<s> $A </s> $B </s>",
+            special_tokens=[
+                ("<s>", ot.convert_tokens_to_ids("<s>")),
+                ("</s>", ot.convert_tokens_to_ids("</s>")),
+            ],
+        )
```

Comment:
oh I'm sorry the change on this file slipped through the cracks when I commited the files.... but indeed this is the fix!

Reply:
@SaulLu I rewrite the converter, this solve the double </s> problem. i.e., the tokenizer.post_processor(). However, the tokenizerfast seem to tokenize "what's his name?" as `["what","'s","his","name","?"]` while as `["what","'s","Ġhis","Ġname","?"]`. Seems the space problem caused by `trim_offsets`?

Reply:
> @SaulLu I rewrite the converter, this solve the double problem. i.e., the tokenizer.post_processor().

:raised_hands: 

> However, the tokenizerfast seem to tokenize "what's his name?" as ["what","'s","his","name","?"] while as ["what","'s","Ġhis","Ġname","?"]. Seems the space problem caused by trim_offsets?

Can you share the snippet you used to see that please?

Reply:
run `test_markuplm_integration_test()`, and you can compare the difference between `encoding_p['input_ids`] and `encoding_r['input_ids']` in Line 2291&2292. I convert the ids into tokens, and gets the results above.

Reply:
I see, it's in the integration test

Reply:
@lockon-n , it's not caused by `trim_offset` is is due to those lines:
https://github.com/NielsRogge/transformers/blob/485b2ca08185231e88d228797e1103ebb43fb23b/src/transformers/models/markuplm/tokenization_markuplm_fast.py#L599-L600

the split delete the spaces

Reply:
Seems removing split() is not ok? i'm not familiar with codes here ...

Reply:
And I realize I hadn't paid attention to this line, but at the moment I don't see a way to correct the problem while still maintaining correct offsets. :disappointed:

Reply:
oh that's quite hard... I'm wondering if we can just give full support to the slow tokenizer in this release?

Reply:
Actually, I'm wondering if this can work:
```python
 if is_pair: 
     batch_text_or_text_pairs = [([text], text_pair) for text, text_pair in batch_text_or_text_pairs] 
```

Reply:
magic! it works indeed, and the test is passed 😄

Reply:
It can do the job :muscle:

Reply:
I think so! @NielsRogge

Reply:
Thanks so much for your effort !!!! 👏 👏  really nice experience to collaborate with such nice staffs, also excited to see our model appearing in this famous repo (a common dream for NLPers I guess 😆)

Reply:
Fantastic! @SaulLu @lockon-n